pyturbo/thermo/ideal_gas.py

```diff
@@ -136,6 +136,11 @@ def f(mach):
 
         return root(f, 0.5).x[0]
 
+    def pcrit_f_pt(self, pt: float, ts: float) -> float:
+        gamma = self.gamma(ts)
+        pcrit = pt * (2 / (gamma + 1)) ** (gamma / (gamma - 1))
+        return pcrit
+
```

Comment:
what is it ?

Reply:
It was a recalculation of critical static pressure, it has been removed because it is useless, the static_p function calculates the same thing when Mach=1